src/inlineStylesheets.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+const fs = require('fs').promises;
+const path = require('path');
+
+let inlineHtmlStyles = async htmlPath => {
+	const linkTagRegex = /<link.+href="([\w.\-\/]+)css".*>/;
```

Comment:
Minor points:
1. Let's enforce the whitespace after 'link'
2. Not enforce a '.css' filename suffix. I'd rather check for ' rel="stylesheet" ' .

```suggestion
	const linkTagRegex = /<link (?:.* )?rel="stylesheet"(?:.* )?href="([\w.\-\/]+)".*>|<link (?:.* )?href="([\w.\-\/]+)"(?:.* )?rel="stylesheet".*>/;
```

3. for consistency, could you likewise add `\-` to the regexes in inlineRequires.js and inlineScriptTags.js?
Thanks for catching this case.